frontend/src/layout.ts

```diff
@@ -56,19 +74,44 @@ export class StatefulLayout {
     }
   }
 
-  nodeColor(eventType: string, hue?: number) {
-    if (!this.colors.has(eventType)) {
-      const hash = [...eventType].reduce(
-        (acc, char) => char.charCodeAt(0) + ((acc << 1) - acc),
-        2
-      )
-      const color = `hsl(${hash % (hue ? hue : 360)},50%,50%)`
-      this.colors.set(eventType, color)
-    }
-    return this.colors.get(eventType)
+  nodeColor(eventType: string) {
+    // if (!this.colors.has(eventType)) {
+    //   const hash = [...eventType].reduce(
+    //     (acc, char) => char.charCodeAt(0) + ((acc << 1) - acc),
+    //     2
+    //   )
+    //   const color = `hsl(${hash % (hue ? hue : 360)},50%,50%)`
+    //   this.colors.set(eventType, color)
+    return this.themeMap.get(eventType).Color
```

Comment:
we need some error handling here in the case event type is not in the theme

Reply:
The person who has been working on this is currently sick but we will forward the feedback so it is worked on when they are feeling better =)